modules/fee/handler.go

```diff
@@ -11,76 +11,67 @@ import (
 // NameFee - namespace for the fee module
 const NameFee = "fee"
 
-// AccountChecker - interface used by SimpleFeeHandler
-type AccountChecker interface {
-	// Get amount checks the current amount
-	GetAmount(store state.KVStore, addr basecoin.Actor) (coin.Coins, error)
-
-	// ChangeAmount modifies the balance by the given amount and returns the new balance
-	// always returns an error if leading to negative balance
-	ChangeAmount(store state.KVStore, addr basecoin.Actor, coins coin.Coins) (coin.Coins, error)
+// Bank is a default location for the fees, but pass anything into
+// the middleware constructor
+var Bank = basecoin.Actor{App: NameFee, Address: []byte("bank")}
```

Comment:
There is no pubkey for it.  It must have authorization from the fee app, which doesn't authorize.

The idea is:
1. SendTx requires a destination (even if it is unusable)
2. A chain can decide to aggregate the fees in some location that is accessible by another app to eg. distribute them to the validators every 1000 blocks.